Tests/TestingApps/PaywallsTester/PaywallsTester/Config/Constants.swift

```diff
@@ -15,7 +15,7 @@ enum Constants {
      REVENUECAT_API_KEY = your-api-key
      */
     static let apiKey: String = {
-        Bundle.main.object(forInfoDictionaryKey: "REVENUECAT_API_KEY") as? String ?? ""
+        "appl_yBXmuJBTosTvPtfwIySkUzDkxAq"
```

Comment:
😱 🤣  trust me, `Local.xcconfig` works 😢

Reply:
Whoops... I hadn't tried it yet, and went this route... I should learn 🥴  Thanks!

Reply:
hahaha no problem! It was just a more fun & constructive way to point out the slipped API key 😆